Dockerfile

```
@@ -0,0 +1,39 @@
+FROM golang:1.11-alpine3.8 as builder
+
+RUN apk add --no-cache --virtual .build-deps gcc musl-dev make git expat
+RUN apk add vips-dev --verbose --update-cache --repository https://dl-3.alpinelinux.org/alpine/edge/testing/ --repository https://dl-3.alpinelinux.org/alpine/edge/main
```

```suggestion
RUN apk add --no-cache \
    --repository https://dl-3.alpinelinux.org/alpine/edge/testing/ \
    --repository https://dl-3.alpinelinux.org/alpine/edge/main \
    vips-dev
```
- Use `--no-cache` to avoid writing to `/var/cache/apk`.
- If you use `--no-cache` no need to `--update-cache`.
- Either add `--verbose` everywhere or nowhere.
- Cosmetic change (line is too long).

---

Ok